db/migrate/20201009160628_add_last_changed_archive_at_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddLastChangedArchiveAtToUser < ActiveRecord::Migration[6.0]
+  def change
+    add_column :users, :last_changed_archive_at, :datetime, default: "2017-01-01 05:00:00"
```

Comment:
I am not a huge fan of adding another "cache" focused column to user but I cant think about a better way to do this. 

What do you think of making this a more general column so we could use it for other caches as well? I was thinking `last_reacted_at` which would fit nicely with our pattern of `last_followed_at`, `last_article_at` etc.

Reply:
Thank you for your comments.

> I am not a huge fan of adding another "cache" focused column to user

Neither am I. There are already a bunch of cache keys.

> What do you think of making this a more general column so we could use it for other caches as well? I was thinking last_reacted_at which would fit nicely with our pattern of last_followed_at, last_article_at etc.

That’s a new idea for me. At least, I think that it is better than mine as we do not need to add a column whenever something related to a user needs to be cached.